pkg/tracer/tracer.go

```diff
@@ -76,11 +91,19 @@ func NewProvider(opts ...Option) (*tracesdk.TracerProvider, error) {
 	} else {
 		//skipped tracing when endpoint no set
 		return nil, nil
-		//trackerTracerProviderOption = append(trackerTracerProviderOption, tracesdk.WithSampler(tracesdk.ParentBased(tracesdk.TraceIDRatioBased(-1))))
 	}
 	if ops.serviceName == "" {
 		ops.serviceName = _service
 	}
+	if ops.samplingRatio != "" {
+		ratio, err := strconv.ParseFloat(ops.samplingRatio, 64)
```

Comment:
Do we need to judge the range of samplingratio here? (ratio < 0, ratio > 1)

Reply:
ratio >= 1 will always sample (default),< 0 are treated as zero will no sample